modules/shared/components/molecules/PostViewHeader/PostViewHeader.tsx

```diff
@@ -1,19 +1,48 @@
 import React from 'react';
 import type { ReactElement, FC } from 'react';
 import Divider from '../../atoms/Divider/Divider';
-import VerticalThreeDotsIcon from '../../icons/verticalThreeDots.svg';
 import type { IPostViewHeader } from './IPostViewHeader';
 import * as EDivider from '../../atoms/Divider/types/EDivider';
 import UserInfo from '../UserInfo/UserInfo';
+import DropDown from '../../atoms/DropDown/DropDown';
+import { options } from '../../atoms/DropDown/mockedOptions';
+
+const getPostOptions = (
```

Comment:
please change the name to **getPostMenuOptions** or a more specific descriptive name.